components/ui/MagicTextArea.tsx

```diff
@@ -75,24 +99,31 @@ const MagicTextArea: FC<Props> = (props: Props) => {
   };
 
   const handleERC20 = (value: string) => {
-    const regex = /^0x[a-fA-F0-9]{40}(?= ?[^ ])([=,]?) *(\d+(\.\d+)?)$/;
+    const regex = /^0x[a-fA-F0-9]{40}(?= ?[^ ])([=,]?) *(\d*(\.\d*)?)$/;
```

Comment:
Why this change?

Reply:
According to @smitch88, "I put up a PR @PopPunkOnChain I also changed the regex so it takes .1 not just 0.*"